doc/lambdapi.bnf

```diff
@@ -153,6 +153,7 @@ QID ::= [UID "."]+ UID
            | "simplify" [<qid_or_nat>]
            | "solve"
            | "symmetry"
+           | TRY <tactic>
```

Comment:
doc/Makefile.bnf should be modified so that TRY is replaced by "try". then do make bnf.